versions.tf

```
@@ -2,7 +2,7 @@ terraform {
   required_version = "~> 0.12.0"
 
   required_providers {
-    aws      = "~> 2.70"
+    aws      = ">= 2.70"
```

Can we cap it at supporting 2.70.x, and 3.x.x so we aren't caught with another major version issue?
```suggestion
    aws = ">= 2.70, < 4.0"
```